PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/HomepageViewModel.kt

```diff
@@ -286,6 +289,25 @@ class HomepageViewModel : HomepageDataModel, ViewModel() {
         })
     }
 
+    private fun getGSRReservations(studentLife: StudentLife, bearerToken: String, latch: CountDownLatch) {
+        studentLife.getGsrReservations(bearerToken).subscribe({ reservationsList ->
+            if (reservationsList.isEmpty()) {
+                addCell(HomeCell(), GSR_POS)
+
+            } else {
+                val gsrCell = GSRCell(reservationsList)
+                Log.i(TAG, "Loaded GSR Reservations")
```

Comment:
Aight

Reply:
This log statement is good methinks but ill check with Aaron